<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VuoMetadataEditor</class>
 <widget class="QDialog" name="VuoMetadataEditor">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>470</y>
     <width>164</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>153</red>
         <green>153</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>153</red>
         <green>153</green>
         <blue>153</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Composition Information</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>541</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 192,192,192</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="VuoLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>351</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>13</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="descriptionLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="VuoLineEdit" name="version">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="versionLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>43</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Version</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="description">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>511</width>
       <height>111</height>
      </rect>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="iconFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget:focus {
border: 2px solid #78a9d2;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="iconLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color: gray</string>
      </property>
      <property name="text">
       <string>Icon for Exported App</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="descriptionDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(92,92,92)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="keywordsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Keywords</string>
     </property>
    </widget>
    <widget class="QLabel" name="keywordsDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>461</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(92,92,92)</string>
     </property>
     <property name="text">
      <string>Comma-separated list of search terms, for when this composition is installed as a node</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="keywords">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab2">
    <attribute name="title">
     <string>License</string>
    </attribute>
    <widget class="QTextEdit" name="copyright">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>441</width>
       <height>61</height>
      </rect>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="licenseText">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>511</width>
       <height>211</height>
      </rect>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="licenseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>84</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>License</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="copyrightLabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>15</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Copyright</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseTitle">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="licenseLinkLabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>343</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Link</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="licenseLink">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>441</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab3">
    <attribute name="title">
     <string>Exporting</string>
    </attribute>
    <widget class="QLabel" name="documentationLinkLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>53</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Documentation Link</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="VuoLineEdit" name="homepageLink">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="VuoLineEdit" name="documentationLink">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="homepageLinkLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>13</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Homepage Link</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="fxPlugGroupLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>153</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>FxPlug Group</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="VuoLineEdit" name="bundleIdentifier">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>90</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bundleIdentifierLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>93</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Bundle Identifier</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="VuoLineEdit" name="fxPlugGroup">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>371</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="bundleIdentifierDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>511</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(92,92,92)</string>
     </property>
     <property name="text">
      <string>A unique name for apps and plugins exported from this composition, in reverse-DNS format.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fxPlugGroupDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>401</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(92,92,92)</string>
     </property>
     <property name="text">
      <string>The category this composition is listed in, when it's exported as an FxPlug.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="categoriesLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Categories</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="categories">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>511</width>
       <height>51</height>
      </rect>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="categoriesDetailsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>461</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(92,92,92)</string>
     </property>
     <property name="text">
      <string>Comma-separated list of Interactive Shader Format (ISF) categories</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VuoLineEdit</class>
   <extends>QLineEdit</extends>
   <header>VuoLineEdit.hh</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>version</tabstop>
  <tabstop>description</tabstop>
  <tabstop>iconFrame</tabstop>
  <tabstop>iconLabel</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>copyright</tabstop>
  <tabstop>licenseTitle</tabstop>
  <tabstop>licenseText</tabstop>
  <tabstop>licenseLink</tabstop>
  <tabstop>homepageLink</tabstop>
  <tabstop>documentationLink</tabstop>
  <tabstop>bundleIdentifier</tabstop>
  <tabstop>fxPlugGroup</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VuoMetadataEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VuoMetadataEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
